assets/scripts/components/charts/ChartSource.tsx

```diff
@@ -1,115 +1,80 @@
-            </div>
-            <div className="fr-collapse" id={targetId} data-fr-js-collapse="true">
-                <h6 className="fr-mt-2w">Source</h6>
-                <p className="fr-text--sm">
-    Données d'évolution des fichiers fonciers produits et diffusés par le Cerema depuis 2009 à 
-    partir des fichiers MAJIC (Mise A Jour de l'Information Cadastrale) de la DGFIP. Le dernier 
-    millésime de 2023 est la photographie du territoire au 1er janvier 2023, intégrant les évolutions 
-    réalisées au cours de l'année 2022.
-    Pour plus d'informations sur <a className="fr-link" href="https://faq.mondiagartif.beta.gouv.fr/fr/article/quest-ce-quun-millesime-110ihlr/" target="_blank" rel="noopener noreferrer">"Qu'est-ce qu'un millésime ?"</a>.
-</p>
+const TagGroup = styled.div`
+    display: flex;
+    align-items: center;
+    gap: 0.5rem;
+`;
+
+const SOURCES_DETAILS: Record<string, { label: string; html: string }> = {
+    insee: {
+        label: 'INSEE',
+        html: `Historique des populations communales issues des recensements de la population (1876-2021) produits et diffusés par l'INSEE.<br/>Les données de population "estimée" ont été réalisées en utilisant une moyenne, permettant de projeter les tendances des années précédentes.`
+    },
+    majic: {
+        label: 'FICHIERS FONCIERS',
+        html: `Données d'évolution des fichiers fonciers produits et diffusés par le Cerema depuis 2009 à partir des fichiers MAJIC (Mise A Jour de l'Information Cadastrale) de la DGFIP. Le dernier millésime de 2023 est la photographie du territoire au 1er janvier 2023, intégrant les évolutions réalisées au cours de l'année 2022.`
+    },
+    gpu: {
+        label: 'GPU',
+        html: `Zonages d'Urbanisme issus du Géoportail de l'Urbanisme (GPU) en date de juin 2023 : <a href='https://www.geoportail-urbanisme.gouv.fr/' target='_blank' rel='noopener'>https://www.geoportail-urbanisme.gouv.fr/</a>`
+    },
+    lovac: {
+        label: 'LOVAC',
+        html: `Base Logements vacants du parc privé par commune, EPCI, département et Région (LOVAC) produite par le Cerema.`
+    },
+    ocsge: {
+        label: 'OCS GE',
+        html: `Données d'OCcupation des Sols à Grande Echelle (OCS GE) de l'IGN.`
+    },
+    rpls: {
+        label: 'RPLS',
+        html: `Répertoire des logements locatifs des bailleurs sociaux (RPLS) produit par le Ministère de la Transition écologique.`
+    },
+    sitadel: {
+        label: 'SITADEL',
+        html: `Base des permis de construire et autres autorisations d'urbanisme (SITADEL) produite par le Ministère de la Transition écologique.`
+    },
+};
 
-                <h6 className="fr-mt-2w">Calcul</h6>
-                <p className="fr-text--sm">Données brutes, sans calcul</p>
+interface ChartSourceProps {
+  sources: string[]; // ['insee', 'majic', 'gpu', 'lovac', 'ocsge', 'rpls', 'sitadel']
+  chartId: string;
+}
```

Comment:
Je crois que tu peux typer avec les clefs d'un objet avec `keyof typeof` (pas testé sur ce cas)

```suggestion
interface ChartSourceProps {
  sources: keyof typeof SOURCES_DETAILS;
  chartId: string;
}
```

Reply:
En effet mais dans notre cas on attend un tableau donc (keyof typeof SOURCES_DETAILS)[];